### ISSUE TITLE
Could PGCRs include the modifiers?

### ISSUE THREAD

**Issue by floatingatoll** [Bungie-net/api#474]
I suspect that, today, the PGCRs do not log the precise modifiers associated with a strike instance. Could they someday include that, so that the historical record is more accurate and so on?

**Comment by vthornheart-bng** [comment]
Indeed, there's no logging of that happening currently - but I think that's a cool idea!

**Comment by jeremyescott** [comment]
I chatted with Edgar on r/DTG, not knowing there was an API GH, so I am leaving a comment here as he requested.

I don't care about the historical record for my own sake, but this information in the API, now and in the future too, could be VERY VERY useful for community-generated content purposes.

I'm discussing making a contest with a friend that will challenge players to achieve feats in the Prestige Nightfall. Each feat must be accomplished with restrictions on guns and/or modifiers. Right now, I'd require the competitors to provide a video to prove they followed the rules regarding modifiers (though PGCR would validate weapons). 

Edgar warned me these things are changed slowly, but this would open doors for us to more easily create user-generated challenges for each other and then compete. Thanks for hearing me out!

**Comment by vthornheart-bng** [comment]
Awesome, thank you for adding your input!  (EdgarVerona here... why I have two aliases is a long story)
